axelrod/player.py

```diff
@@ -28,14 +28,22 @@ class Player(object):
     """
 
     name = "Player"
+    behaviour = {}
 
     def __init__(self):
         """Initiates an empty history and 0 score for a player."""
         self.history = []
-        self.stochastic = "random" in inspect.getsource(self.__class__)
-        self.tournament_attributes = {'length': -1, 'game': None}
+        self.behaviour['stochastic'] = (
+            "random" in inspect.getsource(self.__class__))
         if self.name == "Player":
-            self.stochastic = False
+            self.behaviour['stochastic'] = False
+        if 'memory_depth' not in self.behaviour:
```

Comment:
Cool, am I correct in saying that to add a new dimension, I would add it in two places:

- the specific player I'm adding
- this default dictionary

Just checking I understand (we'll want to document that at some point).